daemon/call.c

```diff
@@ -208,6 +205,12 @@ static void call_timer_iterator(void *key, void *val, void *ptr) {
 		goto delete;
 	}
 
+	// other timeouts not applicable to foreign calls
+	if (IS_FOREIGN_CALL(c)) {
+		c->deleted = c->ml_deleted = poller_now + cm->conf.delete_delay;
```

Comment:
I'm wondering about this addition. This always marks all foreign calls as deleted and keeps moving the deletion timestamp into the future (or into the present, as delete_delay can be 0). What's the reasoning behind this?